docs/discover/search.asciidoc

```diff
@@ -107,23 +107,23 @@ Kibana see <<xpack-security-authorization>>.
 [role="screenshot"]
 image::discover/images/read-only-badge.png[Example of Discover's read only access indicator in Kibana's header]
 
-==== Save a search
-To save the current search:
+==== Save a Discover Session
+To save the current session:
 
 . Click *Save* in the toolbar.
-. Enter a name for the search and click *Save*.
+. Enter a name for the session and click *Save*.
 
-To import, export, and delete saved searches, go to the *Saved Objects* management page using the navigation menu or the <<kibana-navigation-search,global search field>>.
+To import, export, and delete saved Discover Sessions, go to the *Saved Objects* management page using the navigation menu or the <<kibana-navigation-search,global search field>>.
 
-==== Open a saved search
-To load a saved search into Discover:
+==== Open a saved Discover Session
+To load a saved session into Discover:
 
 . Click *Open* in the toolbar.
-. Select the search you want to open.
+. Select the session you want to open.
 
-If the saved search is associated with a different index pattern than is currently
-selected, opening the saved search changes the selected index pattern. The query language
-used for the saved search will also be automatically selected.
+If the saved Discover Session is associated with a different index pattern than is currently
+selected, opening the saved Discover Session changes the selected index pattern. The query language
+used for the saved Discover Session will also be automatically selected.
```

Comment:
```suggestion
If the saved Discover Session is associated with a different {data-source} than is currently
selected, opening the saved Discover Session changes the selected {data-source}. The query language
used for the saved Discover Session will also be automatically selected.
```
Just some more index patterns.

I'm also not sure why we have the same content here as on another page, but also a separate issue.